### PR TITLE
Fix for petco.com

### PR DESCRIPTION
**Pull request by hjpoe**
Invert black logo and menu button (and menu close button) on a dark blue background

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -11059,6 +11059,17 @@ img[src*="logo"]
 
 ================================
 
+petco.com/shop/en/petcostore
+
+INVERT
+a.LogoAnchor__Container-xn7shq-2.ioWQHY
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11065]:
```suggestion
a[class^="LogoAnchor__Container"]
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -11059,6 +11059,17 @@ img[src*="logo"]
...
+button.HamburgerButton-sc-131t62c-0
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11066]:
```suggestion
button[class^="HamburgerButton"]
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -3883,6 +3883,8 @@ developers.google.com
 
 INVERT
 .devsite-google-wordmark
+a.devsite-nav-title.devsite-nav-active
+div.devsite-article-meta.nocontent
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3887]: These are not relevant to this PR.

Comment (hjpoe) [reply src/config/dynamic-theme-fixes.config:3887]: Yeah, I pushed then from GitHub Desktop forgetting that they would go into the same pull request automatically. 😅 We can delete them or something and I'll do a different pull request for them some other time. I won't be available much today unfortunately.

Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3887]:
```suggestion
```
No worries.

**Review by Gusted — APPROVED** [review]

**Pull request merged**